src/index.html

```diff
@@ -22,6 +22,347 @@
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#section-6"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#section-7"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#section-8"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="products">
+      <section
+        class="products__card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="IMAC image"
+          class="products__card_image"
+        />
+        <h2 class="products__card_name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="products__card_product-code">Product code: 195434</h3>
+        <div class="products__card_review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="products__card_review-count">Reviews: 5</div>
+        </div>
+
+        <p class="products__card_price">
+          <span class="products__card_price-text">Price</span>
+          <span class="products__card_price-value">$2,199</span>
+        </p>
+        <a
+          class="products__card_buy-link"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
+      <section class="products__card">
+        <img
+          src="images/imac.jpeg"
+          alt="IMAC image"
+          class="products__card_image"
+        />
+        <h2 class="products__card_name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="products__card_product-code">Product code: 195434</h3>
+        <div class="products__card_review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="products__card_review-count">Reviews: 5</div>
+        </div>
+
+        <p class="products__card_price">
+          <span class="products__card_price-text">Price</span>
+          <span class="products__card_price-value">$2,199</span>
+        </p>
+        <a
+          class="products__card_buy-link"
+          href="#"
+        >
```

Comment:
The `href` attribute with `#` is a placeholder. If this link is not leading anywhere yet, it's better to use `javascript:void(0);` to avoid changing the URL when clicked.